$NetBSD: patch-util_log_c,v 1.1 2012/06/16 05:34:31 dholland Exp $

Provide set_debug_log_from_config() when DEBUG is turned off; avoids
link failure.

--- util/log.c~	2012-05-04 14:51:08.000000000 +0000
+++ util/log.c
@@ -263,4 +263,7 @@ void debug_log(const char *file, int lin
 
 void set_debug_flag(int v)
 { }
+
+int set_debug_log_from_config(config_t c)
+{ return 0; }
 #endif
